### PR TITLE
update-rc.d: add page

### PR DESCRIPTION
**Pull request by jsonbruce**
> update-rc.d - install and remove System-V style init script links

### CODE REVIEW

File: pages/linux/update-rc.d.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# update-rc.d
+
+> Install and remove System-V style init script links
+
+- Installing a service:
+
+ `update-rc.d {{name}} defaults`
```
Comment (luk3yx) [inline pages/linux/update-rc.d.md]: There is a space before the backtick.

File: pages/linux/update-rc.d.md [hunk 2]
```diff
@@ -4,15 +4,15 @@
```
Comment (luk3yx) [inline pages/linux/update-rc.d.md]: There seems to be spaces here.

File: pages/linux/update-rc.d.md [hunk 3]
```diff
@@ -4,15 +4,15 @@
 
 - Installing a service:
 
- `update-rc.d {{name}} defaults`
+`update-rc.d {{name}} defaults`
 
 - Removing a service forcibly:
```
Comment (luk3yx) [inline pages/linux/update-rc.d.md]: ...and before these.

**Comment by waldyrious** [comment]
Thanks for the review @luk3yx :+1: There was another syntax issue -- a missing period at the end of the main description. I fixed it just now.

@jsonbruce I also tweaked the example descriptions to use the infinitive verb tense, as we normally do for tldr-pages. I'm just unsure about the order -- wouldn't it make more sense to use the sequence `install > enable > disable > remove`?

Another question -- what happens if you use `remove` without the `-f`?

File: pages/linux/update-rc.d.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# update-rc.d
+
+> Install and remove System-V style init script links.
```
Comment (agnivade) [inline pages/linux/update-rc.d.md]: What does it mean when you say "script links" ? Is it path to an init script ?

File: pages/linux/update-rc.d.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+- Install a service:
+
+`update-rc.d {{name}} defaults`
```
Comment (agnivade) [inline pages/linux/update-rc.d.md]: What does "name" signify here ? Just the name of the service ? If so, how does it know what is the binary to run and all the service specific stuff ?

File: pages/linux/update-rc.d.md (hunk 4 above)
Comment (sbrl) [inline pages/linux/update-rc.d.md]: I think @jsonbruce is referring to symlinks - correct me if I'm wrong

File: pages/linux/update-rc.d.md (hunk 5 above)
Comment (jsonbruce) [inline pages/linux/update-rc.d.md]: The "name" should be the "/etc/init.d/name "

File: pages/linux/update-rc.d.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# update-rc.d
+
+> Install and remove service.
```
Comment (sbrl) [inline pages/linux/update-rc.d.md]: This doesn't make grammatical sense. Try "services" instead.

Also, what init systems does this work with? Can I use it with `systemd`, or do I need `upstart`? Perhaps it's actually `openrc` that this interacts with. Including this information can avoid confusion.

File: pages/linux/update-rc.d.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+`update-rc.d mysql defaults`
+
+- Forcibly remove a service:
+
+`update-rc.d -f mysql remove`
+
+- Disable a service:
+
+`update-rc.d mysql disable`
```
Comment (sbrl) [inline pages/linux/update-rc.d.md]: Wouldn't I want to enable a service before I disable it? I'd say put enabling before disabling.

File: pages/linux/update-rc.d.md (hunk 6 above)
Comment (agnivade) [inline pages/linux/update-rc.d.md]: Agree that the init system should be clarified.

File: pages/linux/update-rc.d.md [hunk 8]
```diff
@@ -1,19 +1,19 @@
 # update-rc.d
 
-> Install and remove System-V style init script links.
+> Install and remove service.
 
 - Install a service:
 
-`update-rc.d {{name}} defaults`
+`update-rc.d mysql defaults`
```
Comment (agnivade) [inline pages/linux/update-rc.d.md]: Please put mysql in tokens(`{}`). And mention that this file should belong in `/etc/init.d/` folder at the top. (Preferably as a second heading line.)

**Comment by agnivade** [comment]
Hey @sbrl  - can you see if your comments have been addressed ? This one has been lying for some time now.

**Comment by sbrl** [comment]
@agnivade Oops! I didn't realise that this was waiting on me. I'll have to pay more attention next time :P Yeah, my comments have indeed been addressed! I'll merge this now :smiley_cat:

**Pull request merged**